server/service/src/vvm/vvm_status_log/insert/validate.rs

```diff
@@ -0,0 +1,49 @@
+use repository::{
+    vvm_status::{
+        vvm_status_log_row::{VVMStatusLogRow, VVMStatusLogRowRepository},
+        vvm_status_row::{VVMStatusRow, VVMStatusRowRepository},
+    },
+    RepositoryError, StockLineRow, StockLineRowRepository, StorageConnection,
+};
+
+use super::{InsertVVMStatusLogError, InsertVVMStatusLogInput};
+
+pub fn validate(
```

Comment:
will need a user check too~

Reply:
user checks? 👀

Reply:
Oh what I swear I started writing code for this 😂. Adding soon